cgroup_top.py

```diff
@@ -214,6 +217,8 @@ def type(self):
             return 'systemd'
         elif path == '/user.slice' or path == '/system.slice' or path.startswith('/system.slice/'):
             return 'systemd'
+        elif regexp_ovz_container.match(path) and path != '/0' and HAS_OPENVZ:
```

Comment:
It cannot be mixed with openvz now - openvz need custom kernel 2.6.32 or 2.6.18 - https://wiki.openvz.org/Download/kernel/rhel6 . 

In path in this time we have only 
```
2015-05-15 19:25:45,214 - DEBUG - My path: /55384
2015-05-15 19:25:45,215 - DEBUG - My path: /0
2015-05-15 19:25:46,237 - DEBUG - My path: /
2015-05-15 19:25:46,237 - DEBUG - My path: /1212
2015-05-15 19:25:46,238 - DEBUG - My path: /55209
2015-05-15 19:25:46,238 - DEBUG - My path: /79661
2015-05-15 19:25:46,238 - DEBUG - My path: /65217
2015-05-15 19:25:46,239 - DEBUG - My path: /19458
2015-05-15 19:25:46,239 - DEBUG - My path: /39048
2015-05-15 19:25:46,240 - DEBUG - My path: /1207
2015-05-15 19:25:46,240 - DEBUG - My path: /19486
2015-05-15 19:25:46,241 - DEBUG - My path: /60019
2015-05-15 19:25:46,241 - DEBUG - My path: /70259
2015-05-15 19:25:46,241 - DEBUG - My path: /1202
2015-05-15 19:25:46,242 - DEBUG - My path: /74433
```
It all number(because CTID can be only number), because I check that it have openvz, it number and not 0(it is HW node).
If it have more full path - we can watch to  /vz/ fragment - 
```
grep cgroup /proc/mounts  | grep -E "blkio|cpuacct"
beancounter /proc/vz/beancounter cgroup rw,relatime,blkio,name=beancounter 0 0
fairsched /proc/vz/fairsched cgroup rw,relatime,cpuacct,cpu,cpuset,name=fairsched 0 0
```